src/components/styles/icons/triangleExclamation.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+interface TriangleExclamationProps {
+  color: string;
+}
+const TriangleExclamationIconWrapper = styled.div`
+  width: 22.5px;
```

Comment:
Do we have any mobile size differences here? Might need a media query

Reply:
all sizes look consistent across the different devices.  Is a media query still required?

Reply:
actually, did you mean size of icon or size of the Alert? I assumed the latter..